src/applications/vaos/tests/appointment-list/components/CanceledAppointmentsList.unit.spec.jsx

```diff
@@ -32,17 +35,18 @@ describe('VAOS <CanceledAppointmentsList>', () => {
   });
   it('should show information without facility name', async () => {
     const startDate = moment.utc();
-    const appointment = getVAAppointmentMock();
-    appointment.attributes = {
-      ...appointment.attributes,
-      startDate: startDate.format(),
-      clinicFriendlyName: 'C&P BEV AUDIO FTC1',
-      facilityId: '983',
-      sta6aid: '983GC',
+    const data = {
+      id: '1234',
+      currentStatus: 'CANCELLED BY CLINIC',
```

Comment:
I would also suggest with pretty much all of these appointments, also setting a `status` property. In some cases, that will set the `currentStatus` for you, and for the rest of the appointments, it should make switching to the next version easier (since vaos service appointments don't return this vista status).